src/get.js

```diff
@@ -64,12 +95,25 @@ function loadScript(url, callback, errorCallback) {
  * @param {*} opts
  * @returns {Promise}
  */
-function get(src, opts) {
+function get(src, type, opts) {
   if (isString(src)) {
+    //if single src
```

Comment:
Comments should always have one space after the delimiter. Also, comments should be well explained sentences. Also note the full stop at the end of sentence (proper english :smile:).
```suggestion
    // if single source file is provided.
```